css/docs.css

```diff
@@ -0,0 +1,159 @@
+/* Global Variables */
+:root {
+  /* color */
+  --header-bg: #1F2937;
+  --light-grey: #F1F3F5;
+  --pure-white: #FFFFFF;
+  --white: #e8e8fd;
+  --black: #18181B;
+  --dark: #20232A;
+  --grey: #E5E7EB;
+  /* spacing */
+  --side-nav-width: 15rem;
+  --header-height: 4.37rem;
+}
+
+/* Default Styling */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.highlight {
+  background-color: var(--grey);
+  font-family: monospace;
+}
+
+pre {
+  padding: 1rem;
+}
+
+code {
+  border-radius: 0.5rem;
+}
+
+.demo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+}
+
+.demo-column {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+}
+
+.demo-column-stretch {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  padding: 1rem;
+}
+
+.demo-div {
+  background-color: var(--grey);
+  height: 15rem;
+  width: 15rem; 
+  margin: 0.5rem;
+}
+
+/* SITE HEADER */
+.site-header {
+  background-color: var(--header-bg);
+  color: var(--white);
+  display: flex;
+  padding: 1rem;
+  align-items: center;
+  margin-left: var(--side-nav-width);
+}
+
+.site-header .site-logo {
+  font-size: 2rem;
+  margin-left: 1rem;
+}
+
+.site-header .site-logo-img {
+  width: 2.5rem;
+}
+
+.site-header .site-header-nav {
+  margin-left: auto;
+}
+
+.site-header-nav a {
+  color: white;
+  text-decoration: none;
+  margin: auto 1rem;
+}
+
+/* SIDE NAVIGATION */
+aside {
+  width: var(--side-nav-width);
+  background-color: var(--light-grey);
+  position: fixed;
+  height: 100%;
+  overflow: auto;
+  top: 0%;
```

Comment:
`top: 0;` is better as you don't need to specify the CSS unit.

Reply:
fixed